dashboard/templates/dashboard/index.html

```diff
@@ -28,9 +28,15 @@
 
                 </ul>
 
-                <a class="nav-link" href='#'>Profile</a>
-                <a class="nav-link" href="{% url 'dashboard:logout' %}">Logout</a>
-
+                <div class="dropdown">
+                    <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenu2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    {{request.session.username}}
+                     </button>
+                    <div class="dropdown-menu dropdown-menu-sm-right" aria-labelledby="dropdown03">
+                        <button class="dropdown-item">Change Password</button>
```

Comment:
The functionality for the changing the password seems to be missing, so if you could please comment on that.

Reply:
> The functionality for the changing the password seems to be missing, so if you could please comment on that.

Did the user story required change password feature be finished in this sprint? I thought it was save for next time

Reply:
I'm not aware of any changes that might have happened, but based solely on the Zenhub description of the acceptance criteria we do have a "change password page will be implemented". That's why I think @ppietraszko would be able to weigh in on this as well.

Reply:
Yes, the change password feature was part of the story as well.